apis/sgv2-restapi/src/main/resources/application.yaml

```diff
@@ -101,13 +132,3 @@ quarkus:
           ui.preauthorizeApiKey('Token', token);
         }
       }
-
-
-# In Dev mode, gRPC server usually running on different virtual IP
-"%dev":
-  quarkus:
-    grpc:
-      clients:
-        bridge:
-          host: 127.0.0.2
-          port: 8091
```

Comment:
@tatu-at-datastax Removed because this config applies only to your local setup.. You can add `application-dev.properties` for you locally and ignore it with git.. This way you are having a setup only for yourself..

See: https://quarkus.io/guides/config-reference#profile-aware-files

Reply:
Ok. The only thing is that ideally out-of-the-box defaults should work for default setup we use? I was assuming this was the default set up. But if things work without this that's great.

Reply:
`127.0.0.2`, how is this default?

Reply:
It was the setting I saw it in earlier instructions and have used locally, suggesting binding of coordinator that way. Not sure what was the original reason.

So I may be mistaken here; if other APIs not using it then probably REST shouldn't either.

What I meant to ask was just "should it just work for the usual settings/workflow" and I think answer is yes.